src/nbl/ui/CCursorControlWin32.cpp

```diff
@@ -15,27 +15,32 @@ namespace nbl::ui
 		GetCursorInfo(&ci);
 		return ci.flags; // returning flags cause they're equal to 0 when the cursor is hidden
 	}
-	void CCursorControlWin32::setPosition(int32_t x, int32_t y)
+	void CCursorControlWin32::setPosition(CCursorControlWin32::SPosition position)
 	{
-		SetCursorPos(x, y);
+		SetCursorPos(position.x, position.y);
 	}
-	void CCursorControlWin32::setPosition(const core::vector2d<int32_t>& pos)
+	void CCursorControlWin32::setRelativePosition(CCursorControlWin32::SRelativePosition position)
 	{
-		SetCursorPos(pos.X, pos.Y);
+		SPosition nativePos;
+		int32_t screenWidth = GetSystemMetrics(SM_CXSCREEN);
+		int32_t screenHeight = GetSystemMetrics(SM_CYSCREEN);
```

Comment:
wrong, relative position is relative to a window, so you need to take in a window